File: ProjetoFinal/EventosAuth/EventosShared/Model/Evento.cs
namespace EventosShared.Model;

public class Evento
{
	public Guid Id { get; set; }
	public string Titulo { get; set; } = "";
	public string Descricao { get; set; } = "";
	public DateTime DataInicio { get; set; }
	public DateTime DataFim { get; set; }
	public int Vagas { get; set; }
	public int VagasUtilizadas { get; set; }
}

File: ProjetoFinal/Eventos/src/main/java/com/leonardoFanck/Eventos/Model/Evento.java
package com.leonardoFanck.Eventos.Model;

import java.util.Date;
import java.util.UUID;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Evento {
    private UUID id;
    private String titulo;
    private String descricao;
    private Date dataInicio;
    private Date dataFim;
    private int vagas;

}
